### ISSUE TITLE
Calculation of average accuracy

### ISSUE THREAD

**Issue by tshu-w** [megagonlabs/starmie#2]
According to the definition of **average accuracy** from [wikipedia](https://en.wikipedia.org/wiki/Evaluation_measures_(information_retrieval)#Average_precision) or [sklearn](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.average_precision_score.html), AP computes the average value of p(r) over the interval from r=0 to r=1, i.e., the area under precision-recall curve.

However, the following code calculates the mean of precision directly, i.e., the area under precision-number curve.
https://github.com/megagonlabs/starmie/blob/00cae4051f5cf3fe95c416af5edf1f48a92bc5cd/checkPrecisionRecall.py#L90-L94

I believe that this should not affect the overall trend of the different methods and the contribution of this paper, just a friendly reminder :)

**Comment by jw-megagon** [comment]
Thank you so much for pointing it out. We directly use the evaluation metrics from the code in another paper which is a baseline method to compare with. We will add more clarification about it and allow users to customize the evaluation metrics when officially releasing this repo later.
